spec/models/step_spec.rb

```diff
@@ -124,4 +124,41 @@
       expect(step_item.broken_links.length).to eql 1
     end
   end
+
+  describe '.request_broken_links' do
+    context 'when there are links in the step' do
+      it 'should have an associated LinkReport with a batch id' do
```

Comment:
This feels like a copy of the test in `spec/models/link_report_spec.rb`. As that test is checking that the LinkReport is saved correctly, the expectation for this test could just be that LinkReport is being called correctly.

Something like:

`expect(step.link_report).to receive(:create_batch)`